### ISSUE TITLE
Add sources for pages

### ISSUE THREAD

**Issue by Nautman** [Pajsen9263/menu#5]
Hi!

I like your collection! 😄 

I was wondering if it would be possible to add sources to the pages that you include in this collection? A link should probably be enough. 📖 

Thanks!

**Comment by Pajsen9263** [comment]
Hi!
I'm working on a download page. You can see the current beta on 404-pages.github.com/download.html

**Comment by Nautman** [comment]
Wow, that's cool and neat!!

However, I'm assuming you didn't create all the 404 pages completely by yourself? In the cases where you have used someone else's code it would be neat to have a link to where you found it! :)

**Comment by Pajsen9263** [comment]
Yeah, most of them comes from codepen. I'm adding so when you're on the download page there is 2 buttons, one that downloads and one that opens the source in a new tab. But before this comes I'm going to remake the start page and download page.
